java/src/jmri/jmrit/display/layoutEditor/LayoutSlip.java

```diff
@@ -1431,7 +975,11 @@ private void disableSML(SignalMast signalMast) {
         InstanceManager.getDefault(jmri.SignalMastLogicManager.class).disableLayoutEditorUse(signalMast);
     }
 
-    Hashtable<Integer, TurnoutState> turnoutStates = new Hashtable<Integer, TurnoutState>(4);
+    HashMap<Integer, TurnoutState> turnoutStates = new LinkedHashMap<>(4);
```

Comment:
Why is this a LinkedHashMap and not just a HashMap?

Reply:
Because HashMap re-orders the items and LinkedHasMap doesn't and the consensus was to preserve the creation order. I'd prefer them alpha-numerically sorted, but that's just me.

Reply:
Oh, these are the slip states (I was thinking of sensors, turnouts, etc.).
HashMap doesn't preserve any order (accept perhaps hash order?) and I wanted the slip editor to display the state in the order slips cycle via the "Test" button. So I switched to a LinkedHashMap to preserve that order.

Reply:
Because PositionableObjects (Sensors, Labels, Markers, etc.) are stored in a HashMap their creation orders aren't maintained (I think they're sorted/saved by hash order). I'd like to ether change this to maintain the creation order (like the other panel items) or just sort the alphanumerically before saving. But that's a different PR.